sdks/java/core/src/main/java/org/apache/beam/sdk/testing/PAssert.java

```diff
@@ -1112,8 +1266,8 @@ public PDone expand(PBegin input) {
           .apply("WindowToken", windowToken)
           .apply(
               "RunChecks",
-              ParDo.of(new SideInputCheckerDoFn<>(checkerFn, actual, site)).withSideInputs(actual));
-
+              ParDo.of(new SideInputCheckerDoFn<>(checkerFn, actual, site)).withSideInputs(actual))
+          .apply("VerifyAssertions", concludeTransform);
```

Comment:
Rather than calling the varibale (and transform) concludeTransform, just call it "verifyAssertions" everywhere?